gallery/build.gradle

```diff
@@ -1,11 +1,10 @@
 plugins {
```

Comment:
I think we can delete bintray plugin

Reply:
Deleted this piece of code and replcaed it with 'classpath' block in project gradle scrypt and 'apply plugin' in 'publishing.gradle'